dagshub/data_engine/model/datasource.py

```diff
@@ -497,6 +564,13 @@ def __init__(self, dataset: Datasource):
         self._multivalue_fields = dataset._get_multivalue_fields()
 
     def update_metadata(self, datapoints: Union[List[str], str], metadata: Dict[str, Any]):
+        """ 
+        Update metadata for the specified datapoints.
+
+        Args:
+            datapoints (Union[List[str], str]): A list of datapoints or a single datapoint URL to update metadata for.
+            metadata (Dict[str, Any]): A dictionary containing metadata key-value pairs to update.
```

Comment:
Would it be possible to add examples of the correct format, since this is confusing?